application/shared-kernel/InfrastructureCore/InfrastructureCoreConfiguration.cs

```diff
@@ -22,8 +22,7 @@ public static class InfrastructureCoreConfiguration
     [UsedImplicitly]
     public static IServiceCollection ConfigureDatabaseContext<T>(
         this IServiceCollection services,
-        IHostApplicationBuilder builder,
-        string connectionName
+        IHostApplicationBuilder builder
     ) where T : DbContext
     {
         var connectionString = builder.Configuration.GetConnectionString("account-management");
```

Comment:
This is a bug I introduced (i fixed it here... but it's not merged: https://github.com/platformplatform/PlatformPlatform/commit/83918c7e4fa669dccc690bf705c611572cb6d657).

`account-managment` should not be hardcoded in Shared Kernel. So instead of removing `connectionName` it should be used.

Reply:
Ah I see. Then I will delete the PR.